.github/workflows/ci.yaml

```diff
@@ -14,10 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: actions/setup-go@v3
+    - uses: actions/setup-go@v4
       with:
         go-version-file: go.mod
-        cache: true
```

Comment:
:memo: Caching is enabled by default in [setup-go@v4](https://github.com/actions/setup-go/releases/tag/v4.0.0), so I removed this line.